AmplifyPlugins/API/AWSAPICategoryPluginIntegrationTests/REST/RESTWithUserPoolIntegrationTests/RESTWithUserPoolIntegrationTests.swift

```diff
@@ -80,8 +65,8 @@ class RESTWithUserPoolIntegrationTests: XCTestCase {
         wait(for: [completeInvoked], timeout: TestCommonConstants.networkTimeout)
     }
 
-    func testGetAPIFailedWithNotAuthenticated() {
-        AuthHelper.signOut()
+    func testGetAPIFailedWithSignedOutError() {
+        signOut()
```

Comment:
User is already signout at this stage, do we need to call it again?

Reply:
ah no need to call it again, i had this when i was debugging the tests, in a normal test run, this is covered already in the `tearDown`. removing this line